src/lib/pages/backoffice/Acquisition/Order/OrderEditor/OrderEditor.js

```diff
@@ -65,7 +65,7 @@ export class OrderEditor extends Component {
     const request = _get(this.props, 'location.state', null);
     if (!request) return null;
     return {
-      documentRequestPid: request.pid,
+      documentRequestPid: request.id,
```

Comment:
did you mean `request.metadata.pid`? `.id` is inconsistent with all the other record types we have

Reply:
I intentionally changed it to `id` because yesterday I tested it and the `pid` was sometimes undefined but the `id` had a value but I probably did something wrong because now that I tested it it's working fine.